recipes/stompy-ocean/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "stompy-ocean" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stompy_ocean-{{ version }}.tar.gz
+  sha256: 536d8ef35fb8db50bc2dc485c41c6e95e2e94be045113bcdd70726c513fa7416
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - cgal-cpp
+  run:
+    - python >=3
+    - cmocean
+    - matplotlib-base
+    - netcdf4
+    - pandas
+    - requests
+    - scipy
+    - shapely
+    - six
+    - xarray
+    - seawater
+    - gdal <3.9
+    - cgal
+    - cgal-cpp
+    - numpy <2.0
+
+test:
```

Comment:
Maybe add a `pip check`?
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

Reply:
I was having issues with the `pip check` saying that `cgal` was not installed, even though it was working fine when I tested locally in different envs. Still trying to troubleshoot that.